diff --git a/dist/utils/get-root-dirs.js b/dist/utils/get-root-dirs.js
index 47108c943514a1d7e686c5eb18aad00a59963c83..5996520708be1a83e35e9999e2b9b579cff6fbc2 100644
--- a/dist/utils/get-root-dirs.js
+++ b/dist/utils/get-root-dirs.js
@@ -13,7 +13,9 @@ var _fastglob = require("fast-glob");
  * Process a Next.js root directory glob.
  */ var processRootDir = function(rootDir) {
     return (0, _fastglob.globSync)(rootDir.replace(/\\/g, '/'), {
-        onlyDirectories: true
+        onlyDirectories: true,
+        expandDirectories: false,
+        absolute: require("node:path").isAbsolute(rootDir)
     });
 };
 var getRootDirs = function(context) {
